node/netprio.go

```diff
@@ -28,10 +28,12 @@ import (
 
 const netPrioChallengeSize = 32
 
+const netPrioChallengeSizeBase64Encoded = 44 // 32 * (4/3) rounded up to nearest multiple of 4 -> 44
```

Comment:
I mentioned this in the resolved conversation above but this is already locked down via:

`require.Len(t, nonce, netPrioChallengeSizeBase64Encoded)`

the nonce there is generated by

`nonce := node.NewPrioChallenge()`

Which uses the `netPrioChallengeSize` but even if it changes what it uses to generate the challenge it will fail. 
Also once again the Base64 encoding is deterministic and this is the actual upperbound for the generated nonce. Base64 encoding is a [standard implementation of an RFC 4648](https://pkg.go.dev/encoding/base64).

Allocbound is also the upperbound so if the implementation changes to not use padding which would result in length of 43 that would still be safe for the protocol (less than the allocbound) and would still be caught by the test as well since it checks for exact length matching the constant.